packages/template-icons/src/useUniqueId.tsx

```diff
@@ -0,0 +1,9 @@
+import { useMemo } from "react";
+
+let counter = 0;
+
+const useUniqueId = () => {
+  return useMemo(() => counter++, []);
+};
+
+export default useUniqueId;
```

Comment:
You should use a named export instead of the default one.

Reply:
Fixed